.circleci/config.yml

```diff
@@ -31,8 +28,7 @@ jobs:
           command: lein test-ci
       - run:
           name: Code Coverage
-          command: lein cloverage --codecov
-      - codecov/upload
+          command: lein cloverage
```

Comment:
I think we'd want to store the artifacts for bond as well.

Reply:
It's already being done at line 36 :)